politeiawww/api/v1/v1.go

```diff
@@ -566,9 +573,10 @@ type ActiveVoteReply struct {
 // plugin commands
 // StartVote starts the voting process for a proposal.
 type StartVote struct {
-	PublicKey string `json:"publickey"` // Key used for signature.
-	Vote      Vote   `json:"vote"`      // Vote
-	Signature string `json:"signature"` // Signature of Votehash
+	PublicKey    string `json:"publickey"`              // Key used for signature.
+	Vote         Vote   `json:"vote"`                   // Vote
+	Signature    string `json:"signature"`              // Signature of Votehash
+	CompactReply bool   `json:"compactreply,omitempty"` // Boolean flag to ask for the compact reply
```

Comment:
We don't want this. We want a new route for all compacts. So the GUI uses the compacts and the voting tools uses the larger ones. In fact once we convert to the compact ones we may be able to delete the non-compact routes and convert the tools to use all compacts as well.